### PR TITLE
SWED-2249 create playbook doc

### PR DESCRIPTION
**Pull request by goldenraphti**
Don't get scared by the amount of files changed. Most of it (160 files) is image & video files, each with their own formats (images = `.png` + `.avif` , and videos are in `.mp4` but in both the classic codec and as `AV1` encoded, for performance reasons)

Also, since the sidebar got a new section, many tests files snapshots and screenshots had to be updated 😛 (3 screenshots)

## Description

add the Playbook section to the DS documentation website
https://payexjira.atlassian.net/browse/SWED-2249

- add Playbook section
- remove Identity Overview page (replace the landing by the Identity/Accessibility

## Types of changes

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

-   [x] My code follows the code style of this project.
-   [ ] My change requires a change to the documentation.
-   [ ] I have updated the documentation accordingly.
-   [x] I have read the **CONTRIBUTING** document.
-   [ ] I have updated the **CHANGELOG** document.
-   [ ] I have added tests to cover my changes.
-   [x] All new and existing tests passed.

## Review instructions

[Review instructions](../REVIEW_INSTRUCTIONS.md)

### CODE REVIEW

File: src/App/Identity/identity/Typography/typography.e2e.spec.js
Comment (goldenraphti) [inline src/App/Identity/identity/Typography/typography.e2e.spec.js:1]: reason of changes: we removed the Overview page from Identity section. Therfore each idenity page only has a link in the sidebar, there is no morelink in the (removed) Overview page. Now landing page for Identity is Accessibility

File: src/App/Home/index.js
Comment (goldenraphti) [inline src/App/Home/index.js:1]: Reason of change:
Playbook is only for SwedbankPay.
So the homepage will still have only 4 quick-access cards. And whether it's SwedbankPay or PayEx one of them will either be Playbook Or Patterns

File: src/App/Playbook/content/Showroom/index.js [hunk 1]
```diff
@@ -0,0 +1,530 @@
...
+				</div>
+			</div>
+			<div className="row mt-3">
+				<div className="col-sm">
+					<picture>
+						{/* <source
+							type="image/avif"
+							media="(min-width: 576px)"
+							srcSet={`${basename}img/documentation/playbook/playbook-showroom-2.avif`}
+						/> */}
```
Comment (goldenraphti) [inline src/App/Playbook/content/Showroom/index.js:42]: Reasoning:
"Why keeping code commented-out?" You want to ask ?
- Well, great question !! (I first added the images in avif too, to get better web-perf. But then designers asked me to yet increase the quality of the images. My issue is that I think the quality is good enough (3x everywhere) and I converted to avif in as high quality as possible and lossless. And with all the img/media content on this page the computers and phones are struggling to process and paint it all fluidly (well, they really struggle on a non 3000$ machine such as the crappy old laptop I am using, and several videos cannot even play). I do suspect this decision will change in the near future in an iteration for the necessity of actually having the page displaying its content fluidely even if it means slightly less quality). So here it is, ready to be un-commented at the snap of a finger 😉🤫

File: src/less/documentation-swedbankpay.less [hunk 2]
```diff
@@ -720,6 +720,183 @@
 			}
 		}
```
Comment (goldenraphti) [inline src/less/documentation-swedbankpay.less:722]: Note of the dev:
I am not 100% happy with all this styling.
I think there is clearly room for simplification, reformatting it to have less "1 case rules".
So far we're a bit in a rush, so here it is.
Plus this is documentation website style only, and not even for the components, so it's (in my opinion) much less important to get it super nice and clean.
But I do want to go through it all again and improve & clean what can be 👍

**Review by MariusSjo — APPROVED** [review]
I think it looks good! 

It is interesting to see the "conflict" on the user's loading time vs quality of the visualisations on the page.

I think it is important to make our product available. However, if the users are developers with good internet connection and fast computers we might not have to have this discussion as prio 1. it would be nice with some guidelines on what to prioritise.

Nice job ✅

**Comment by goldenraphti** [comment]
bypassing approval since it was approved my Marius, and only had to modify the release-notes markdown for merge-conflict 😉

**Pull request merged**